saundracatalina/crate-1
# Test testing

### What does this PR do? 
Adds testing configuration setup for BE part of app

### Where should the reviewer start?
code/api/package.json
code/api/src/config/database.json

### How should this be tested?
We wrote a sample test to make sure test setup is functional and did not break anything. 

### This PR is related to which issues:
Closes #23
